tlsconfig/tlsconfig.go

```diff
@@ -130,6 +130,14 @@ func Default(config *tls.Config) {
 	config.PreferServerCipherSuites = true
 }
 
+// DefaultCommonRuntime modifies config with safe defaults for standard compatibility.
+// This is different from Default because the common runtime supports TLS 1.0
```

Comment:
"supports TLS 1.0 [until 2021-07-31](https://help.heroku.com/SXWA00YI/tls-v1-0-v1-1-end-of-life-schedule)" to make it explicit that this is going away very soon?